.github/workflows/component-bumper.yaml

```diff
@@ -14,11 +14,10 @@ jobs:
       matrix:
         branch:
           - main
-          - release-0.53
-          - release-0.58
           - release-0.65
```

Comment:
nit:
maybe worth to add sanity check that RELEASES_SELECTOR lower than oldest doesn't appear here ?
and that all that do appear there appear here as well ?

Reply:
That's a connection between D/S and U/S that I'm not sure we want to have

Reply:
implicitly yes, but both are on U/S
anyhow whatever you prefer